pages/common/ppmtopgm.md

```diff
@@ -0,0 +1,12 @@
+# ppmtopgm
+
+> Convert a PPM image to a PGM image.
+> More information: <https://netpbm.sourceforge.net/doc/ppmtopgm.html>.
+
+- Generate the PGM file as output, for a PPM file as input:
+
+`ppmtopgm {{path/to/file.ppm}}`
```

Comment:
```suggestion
`ppmtopgm {{path/to/file.ppm}} > {{path/to/file.pgm}}`
```

For consistency, because we also have it that way in other netpgm pages.